Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -399,6 +399,17 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.NONE,
         },
+        [78270] = {
+            [questKeys.name] = "Goblin Lockpicks",
+            [questKeys.startedBy] = {{210187}},
```

Comment:
The first entry of the `startedBy` table is a table of NPC IDs which start a quest. This can either be friendly/neutral NPCs to which the player has to talk or hostile NPCs from which the player has to loot an item, which starts the quest.

In this case you want it to be the NPCs which "drop" the `Venture Co. Work Order`.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5339/commits/3197ef226acfa58c0d5e76bd3c75d0cb3cb9fd5f